create_project_workspace.py

```diff
@@ -68,11 +74,15 @@ def main(argv=None):
     parser = argparse.ArgumentParser(description="Bazel Federation WORKSPACE Generation Script")
     parser.add_argument("--project", type=str, required=True)
     parser.add_argument("--repo", type=str)
+    parser.add_argument("--has_internal_deps", type=bool)
```

Comment:
I'd prefer if has_internal_deps was the default case, and non-compliant projects would have to set an explicit --no_internal_deps flag.